### ISSUE TITLE
Training loss data

### ISSUE THREAD

**Issue by borgr** [mlfoundations/scaling#4]
I have found in the git the last evals, but do you have somewhere the loss\evals during training and not only the last step?
Could you please share it?

**Comment by sagadre** [comment]
hi @borgr, thanks for the interest in the work! just did a PR to add support for downloading the logs that we have. see this notebook: `training_explorer.ipynb`

**Comment by borgr** [comment]
Oh, that looks great, thanks.
Maybe should write there
 the "legend" for the names (name,d,l,h) should make people's life a tad easier.
Also, here it is in csv that you can load like 
pd.read_csv("overtraining.csv")

[overtraining.csv](https://github.com/mlfoundations/scaling/files/14843771/overtraining.csv)

**Comment by sagadre** [comment]
see this constant to get the friendly names: https://github.com/mlfoundations/scaling/blob/a003c4913793ac2ae7ef87b28ecb562955d026d5/scaling/constants.py#L139-L146

closing the issue for now, but feel free to let me know if you have more questions!
